lib/mayaUsd/python/wrapEditRouter.cpp

```diff
@@ -74,4 +74,9 @@ void wrapEditRouter()
             return MayaUsd::registerEditRouter(
                 operation, std::make_shared<PyEditRouter>(editRouter));
         });
+
+    def(
+        "restoreDefaultEditRouter", +[](const PXR_NS::TfToken& operation) {
```

Comment:
Why not just setting the address of the function to call instead of creating a lambda function that is calling the function directly?

Reply:
Quite right!  I just mindlessly copy-pasted the code above, there's no need for a lambda in this case, good catch.